### PR TITLE
Install and Secure Nginx with Let's Encrypt in Debian 10

### PR DESCRIPTION
**Pull request by Masteritx**
I have read and understood the Contributor's Certificate of Origin available at the end of
https://raw.githubusercontent.com/hetzneronline/community-content/master/tutorial-template.md
and I hereby certify that I meet the contribution criteria described in it.
Signed-off-by: David Masteritx <8766@protonmail.com>

### CODE REVIEW

File: tutorials/install-and-secure-nginx-lets-encrypt-debian-10/01.en.md [hunk 1]
```diff
@@ -0,0 +1,342 @@
...
+
+Nginx web service is one of the most popular web servers in the world and is used as a web service in large and highly visited websites.
+Let’s Encrypt is an SSL or CA security certificate issuer that is free to install on the web server and use as an HTTPS protocol.
+In this tutorial, we will learn how to install Nginx on Debian 10, how to install Let’s Encrypt as an SSL certificate issuer, and how to automatically update SSL.
+
+**Prerequisites**
+
+* Make sure you are logged in with a user with sudo access to the server.
+* A registered domain with the required records as follows:
+   * A record with `your_domain` and a reference to the server's IP address
```
Comment (n-se) [inline tutorials/install-and-secure-nginx-lets-encrypt-debian-10/01.en.md:28]: Please use the example terminology from the template https://github.com/hetzneronline/community-content/blob/master/tutorial-template.md

File: tutorials/install-and-secure-nginx-lets-encrypt-debian-10/01.en.md [hunk 2]
```diff
@@ -0,0 +1,342 @@
...
+
+```bash
+nginx: the configuration file /etc/nginx/nginx.conf syntax is ok
+nginx: configuration file /etc/nginx/nginx.conf test is successful
+```
+
+After configuring, restart Nginx to apply the changes.
+
+```bash
+sudo systemctl restart nginx
```
Comment (n-se) [inline tutorials/install-and-secure-nginx-lets-encrypt-debian-10/01.en.md:180]: reload is enough

File: tutorials/install-and-secure-nginx-lets-encrypt-debian-10/01.en.md [hunk 3]
```diff
@@ -0,0 +1,342 @@
...
+sudo apt install python3-acme python3-certbot python3-mock python3-openssl python3-pkg-resources python3-pyparsing python3-zope.interface
+```
+
+Then install the `python3-certbot-nginx` package with the following command.
+
+```bash
+sudo apt install python3-certbot-nginx
+```
+
+## Step 5 - Confirm Nginx configuration
```
Comment (n-se) [inline tutorials/install-and-secure-nginx-lets-encrypt-debian-10/01.en.md:207]: I dont understand this step. We just create a new config with server_name and now we validate again if we did it correctly? That does not make much sense to me.

File: tutorials/install-and-secure-nginx-lets-encrypt-debian-10/01.en.md [hunk 4]
```diff
@@ -0,0 +1,342 @@
...
+
+   Donating to ISRG / Let's Encrypt:   https://letsencrypt.org/donate
+   Donating to EFF:                    https://eff.org/donate-le
+```
+
+## Step 7 - Automatic renewal of SSL certificate
+
+Let’s Encrypt certification is only valid for 90 days. This has led users to need to automatically renew their security certification.
+
+For this purpose, by adding the extension script to `/etc/cron.d`, we can perform the renewal process automatically.
```
Comment (n-se) [inline tutorials/install-and-secure-nginx-lets-encrypt-debian-10/01.en.md:297]: Okay we can do that, but how can we do that? :) I think the steps are missing here

File: tutorials/install-and-secure-nginx-lets-encrypt-debian-10/01.en.md (hunk 1 above)
Comment (Masteritx) [reply tutorials/install-and-secure-nginx-lets-encrypt-debian-10/01.en.md]: Fixed.

File: tutorials/install-and-secure-nginx-lets-encrypt-debian-10/01.en.md (hunk 2 above)
Comment (Masteritx) [reply tutorials/install-and-secure-nginx-lets-encrypt-debian-10/01.en.md]: Fixed.

File: tutorials/install-and-secure-nginx-lets-encrypt-debian-10/01.en.md (hunk 3 above)
Comment (Masteritx) [reply tutorials/install-and-secure-nginx-lets-encrypt-debian-10/01.en.md]: Step 5 has been removed.

File: tutorials/install-and-secure-nginx-lets-encrypt-debian-10/01.en.md (hunk 4 above)
Comment (Masteritx) [reply tutorials/install-and-secure-nginx-lets-encrypt-debian-10/01.en.md:265]: Check it again.

Step 6 has been edited and Explain more about renewal SSL.

**Pull request merged**